kairon/api/app/routers/bot/bot.py

```
@@ -1700,3 +1701,92 @@ async def get_flow_tag(
+    result = DataProcessor.update_crud_collection_data(
+        collection_id=payload["id"],
+        user=current_user.get_user(),
+        payload=payload,
+        bot=current_user.get_bot()
+    )
+    return Response(data=result, message="Doccument Updated")
+
+
+@router.delete("/collections/data/delete", response_model=Response)
+async def delete_collection_data(
+    payload: dict = Body(...),
```

Ruff (0.11.9)

Do not perform function call `Body` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)